roles/nginxplus/vars/proxies_by_box.yml

```diff
@@ -0,0 +1,141 @@
+# list of proxy directives
```

Comment:
@acozine the only concept I am not seeing is that we have different proxies for the staging library site and the production library site.  For example: https://library.princeton.edu/njmaps/ and https://library-staging.princeton.edu/njmaps/. go to two different places  https://lib-static-prod.princeton.edu/njmaps/ for prod and https://firestone-renovations-staging.princeton.edu/njmaps/ for staging.  This is pattern is used a lot to proxy the staging version of the page to library-staging.princeton.edu and the production version to library.princeton.edu.  This is necessary to test how the page will behave behind the proxy before putting changes into production.

Reply:
I know this is a WIP, so I am just making sure you have that in mind...

Reply:
Ah, thanks @carolyncole. I will make two files here - one for prod and one for staging.